test/mockApiServer.ts

```diff
@@ -1,15 +1,30 @@
 import * as express from 'express';
 
-import { MOCK_UFC_RESPONSE_FILE, readMockUFCResponse } from './testHelpers';
+import {
+  MOCK_BANDIT_MODELS_RESPONSE_FILE,
+  MOCK_FLAGS_WITH_BANDITS_RESPONSE_FILE,
+  MOCK_UFC_RESPONSE_FILE,
+  readMockResponse,
+} from './testHelpers';
 
 const api = express();
 
 export const TEST_SERVER_PORT = 4123;
+export const TEST_BANDIT_API_KEY = 'bandit-test-key';
 const flagEndpoint = /flag-config\/v1\/config*/;
+const banditEndpoint = /flag-config\/v1\/bandits*/;
 
-api.get(flagEndpoint, (_req, res) => {
-  const mockRacResponse = readMockUFCResponse(MOCK_UFC_RESPONSE_FILE);
-  res.json(mockRacResponse);
+api.get(flagEndpoint, (req, res) => {
+  const ufcFile = req.url.includes(TEST_BANDIT_API_KEY)
+    ? MOCK_FLAGS_WITH_BANDITS_RESPONSE_FILE
+    : MOCK_UFC_RESPONSE_FILE;
```

Comment:
Some tests use one file, some use the other; use the API key to decide which to serve